kotlin/workflow-ui-android/src/main/java/com/squareup/workflow/ui/ViewShowRendering.kt

```diff
@@ -60,8 +67,8 @@ fun View.canShowRendering(rendering: Any): Boolean {
 fun <RenderingT : Any> View.showRendering(rendering: RenderingT) {
   showRenderingTag
       ?.let { tag ->
-        check(tag.showing.matches(rendering)) {
-          "Expected $this to be able to update of ${tag.showing} from $rendering"
+        check(tag.initialRendering.matches(rendering)) {
```

Comment:
I want to follow up and start updating this value. In the meantime, `initialRendering` is a more accurate description of how things actually work at the moment.